.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,16 +3,23 @@ _List Changes Introduced by this PR_
 1. Item 1
 2. Item 2
 
+## Purpose
+_Describe the problem or feature in addition to a link to the issues._
+
 ## Approach
 _How does this change address the problem?_
 
+## Pre-Testing TODOs
+_What needs to be done before testing_
+- [] Change DB values in x collection to y.
```

Comment:
I know its silly and pedantic but how do you feel about changing `DB` to `Database`? Its a bit more clear. 

Really new developers will not know the acronyms so this may help with a very small amount of confusion. At least that is the hope.

Reply:
Thanks!